src/Modules/Guilds/Hooks/useGuildHolderTotal.ts

```diff
@@ -0,0 +1,13 @@
+import ERC20SnapshotRep from 'contracts/ERC20SnapshotRep.json';
+import { useContractRead } from 'wagmi';
+
+const useGuildHolderTotal = (guildToken: string) => {

```

Comment:
❓ The naming is a bit weird here. What about calling it `useTotalGuildMembers` instead or something else that's a bit more descriptive?